app.py

```diff
@@ -132,23 +133,20 @@ def _event_producer(queue: Queue, build_watcher_namespace: str):
             strategy = event["object"].spec.strategy
             build_reference = _get_build(openshift, strategy, build_reference, event["object"].metadata)
             _LOGGER.info("Queueing build log based on build event %r for further processing", event_name)
-            _METRIC_IMAGES_SUBMITTED.inc()
             _METRIC_BUILDS_FAILED.inc()
+            try:
+                _LOGGER.info(f"Submitting metrics to Prometheus pushgateway {_THOTH_METRICS_PUSHGATEWAY_URL}")
```

Comment:
It's better to expand logged message lazily based on loglevel set (f-strings will be always expanded):

```python
                _LOGGER.info(f"Submitting metrics to Prometheus pushgateway %r", _THOTH_METRICS_PUSHGATEWAY_URL)
```

Here and bellow.